deployment/terraform/storage.tf

```diff
@@ -17,7 +17,29 @@ resource "aws_s3_bucket" "static" {
 
 resource "aws_s3_bucket" "storage" {
   bucket = "${lower("${var.environment}")}-pfb-storage-${var.aws_region}"
-  acl    = "private"
+  acl    = "public-read"
+  policy = <<POLICY
```

Comment:
`var.bucket_name` was something that came along for the ride from the example I pulled. I think that it can safely be replaced with `arn:aws:s3:::${lower("${var.environment}")}-pfb-storage-${var.aws_region}/*` here.

Associating it with the bucket would be something like:

```hcl
resource "aws_s3_bucket" "storage" {
  bucket = "${lower("${var.environment}")}-pfb-storage-${var.aws_region}"
  policy = "${data.aws_iam_policy_document.read_only_bucket_policy.json}"

  ...
}
```

First time seeing `aws_s3_bucket_policy`, actually. I'd lean toward rending the policy JSON directly into the `policy` parameter of `aws_s3_bucket`.